### PR TITLE
F vue3 async components

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T19785
https://v3-migration.vuejs.org/breaking-changes/async-components.html

- add defineAsyncComponent to async components.
- New defineAsyncComponent helper method that explicitly defines async components
- Async component without options

### CODE REVIEW

File: client/js/bundles/procedure/administrationProcedureTypeEdit.js [hunk 1]
```diff
@@ -15,15 +15,13 @@
...
 import { dpValidate } from '@demos-europe/demosplan-ui'
 import { initialize } from '@DpJs/InitVue'
 import ProcedureTypeSelect from '@DpJs/components/procedure/admin/ProcedureTypeSelect'
 
 const components = {
-  DpEditor: async () => {
-    const { DpEditor } = await import('@demos-europe/demosplan-ui')
-    return DpEditor
-  },
+  DpEditor: defineAsyncComponent(() => import('@demos-europe/demosplan-ui')),
```
Comment (hwiem) [inline client/js/bundles/procedure/administrationProcedureTypeEdit.js:24]: Are you sure that this works? Shouldn't you import DpEditor specifically?

Comment (ahmad-demos) [reply client/js/bundles/procedure/administrationProcedureTypeEdit.js]: yes, done in https://github.com/demos-europe/demosplan-core/pull/2424/commits/160ca655c6617d9427f3aea0e97f0b87c8b4b677, thnx!

Comment (ahmad-demos) [reply client/js/bundles/procedure/administrationProcedureTypeEdit.js]: Thanks
Done in https://github.com/demos-europe/demosplan-core/pull/2424/commits/3dbc0f8e5ea38fb9d49214443c10cf6b699b6046

**Review by hwiem — APPROVED** [review]
lgtm, code review only. thanks for fixing this!

**Pull request merged**